src/components/backup/BackupManualStep.js

```diff
@@ -28,7 +28,6 @@ const Content = styled(Column).attrs({
 `;
 
 const Footer = styled(Column).attrs({
-  align: 'center',
```

Comment:
I removed this so that button in the footer uses the default `stretch` alignment, causing it to be full width. The button is the only element in the footer so `center` alignment was unnecessary here.